### PR TITLE
WM4 - Hayatullahi Adeyemo - Full Stack Project Assessment - Week 1

### PR DESCRIPTION
**Pull request by hayatu4islam**

### CODE REVIEW

**Review by Dedekind561 — CHANGES_REQUESTED** [review]
Hey @hayatu4islam, 
How's it going?
It looks like you've not completed any of the requirements for Week 1. Please make sure you fulfil the coursework requirements before opening a PR for review. Remember to ask for support on Slack if you're stuck with the requirements too.

File: client/src/App.js [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <header className="my-App-header">
         <h1>Video Recommendation</h1>
       </header>
+      <body>
+          {defaultVideos.map((video, key) => (
```
Comment (Know-Thyself) [inline client/src/App.js:13]: Rendering the data you imported directly on your page is not a good idea as you will need to manipulate/update it at some point based on the user interaction. Storing the data in a react state variable is your best bet.

File: client/src/InsertVideo.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Delete from "./clickables/Delete";
+
+export default function InsertVideo({video}) {
+    const src = video.url.replace("watch?v=", "embed/")
```
Comment (Know-Thyself) [inline client/src/InsertVideo.js:5]: Nice way of embedding a YouTube video without having to extract an id from a url!
How did you come up with this solution?

**Review by Know-Thyself — COMMENTED** [review]
Good start but you still have a lot to do to meet the requirements of this assessment.
I'll be looking forward to see more of your work.

File: client/src/InsertVideo.js (hunk 2 above)
Comment (hayatu4islam) [reply client/src/InsertVideo.js:5]: I realize embedding the url is not working and I checked samples of embedded YouTube url and realised the difference.

File: client/src/App.js (hunk 1 above)
Comment (hayatu4islam) [reply client/src/App.js]: I have completed level 100. Can you please review it? Thanks

**Pull request closed without merging**